src/modifyTodo.js

```diff
@@ -0,0 +1,89 @@
+const data = { index: 1, description: '', completed: false };
+
+export const addTodo = (todosFromLocalStorage) => {
+  const inputField = document.querySelector('#new-todo');
+  const addBtn = document.querySelector('.add-btn');
+
+  inputField.addEventListener('change', (event) => {
+    data.description = event.target.value;
+  });
+
+  addBtn.addEventListener('click', () => {
+    todosFromLocalStorage.push(data);
+    todosFromLocalStorage[todosFromLocalStorage.length - 1].index = todosFromLocalStorage.length;
+    localStorage.setItem('todos', JSON.stringify(todosFromLocalStorage));
+    window.location.reload();
+  });
+};
+
+export const deleteTodo = (todosFromLocalStorage) => {
+  const deleteIcons = document.querySelectorAll('.delete-todo');
+  let listOfTodos = [];
+  deleteIcons.forEach((icon) => {
+    icon.addEventListener('click', () => {
+      const todoId = Number(icon.parentElement.id);
+      todosFromLocalStorage.forEach((todo) => {
+        if (todoId === todo.index) {
+          listOfTodos = todosFromLocalStorage.filter(
+            (todo) => todo.index !== todoId,
+          );
+        }
+      });
+      listOfTodos.forEach((todo, index) => {
+        todo.index = index + 1;
+      });
+      localStorage.setItem('todos', JSON.stringify(listOfTodos));
+      window.location.reload();
```

Comment:
Instead of writing it like this, just create a function that you can call in one line to reduce the code.